src/components/Booking/StatusBar/index.js

```diff
@@ -61,7 +61,7 @@ export default StatusBar;
 
 StatusBar.propTypes = {
   booking: PT.shape({
-    holId: PT.number,
+    eventID: PT.number,
```

Comment:
super petty, but can we make `eventID` `eventId` throughout this PR? - then I'm happy to approve

Reply:
I made the changes :)